src/styles/blocks/card.scss

```diff
@@ -0,0 +1,133 @@
+.container {
+  display: grid;
+  justify-content: center;
+  grid-gap: 46px 48px;
+  padding: 50px 40px;
+
+  @include grid;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  font-family: inherit;
+
+  border-radius: 5px;
+  border: 1px solid $dark-white-color;
+
+  box-sizing: border-box;
+
+  transition: transform $transition, color 1s;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__logo {
+    height: 134px;
+    width: 160px;
+  }
+
+  &__title {
+    margin: 40px 0 0;
+
+    font-family: inherit;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+
+    font-size: 10px;
+    font-style: normal;
```

Comment:
`font-style` by default is `normal`, so you don't need to set it manually, fix it in all file.